pkg/providers/launchtemplate/suite_test.go

```diff
@@ -1380,6 +1386,188 @@ var _ = Describe("LaunchTemplates", func() {
 				ExpectLaunchTemplatesCreatedWithUserData(expectedUserData)
 			})
 		})
+		Context("AL2023", func() {
```

Comment:
Can users override kubeletConifuration using the userdata? If so, should we test that?

Reply:
That's part of the merge tests. We don't need to handle override logic on our side since `nodeadm` will merge multiple `NodeConfig`s from top to bottom, with later `NodeConfig`s taking precedent.